pypesto/visualize/sampling.py

```diff
@@ -1,7 +1,8 @@
 import logging
 import warnings
+from collections.abc import Sequence
```

Comment:
Any particular reason to use this one? Seems like just an extra import to me.

Reply:
Yes. The former is deprecated since python 3.9.